config/install/field.storage.user.field_age_group.yml

```diff
@@ -11,20 +11,29 @@ type: list_string
 settings:
   allowed_values:
     -
-      value: '19'
-      label: 15-19
+      value: '1939'
```

Comment:
I still have a feeling that field type should be `list_integer` instead of `list_string`